assignments/assignment01.rb

```diff
@@ -80,8 +77,14 @@ def my_reverse(s)
 # Write a method `palindrome?`
 # that determines whether a string is a palindrome
 def palindrome?(s)
-  stripped = s.delete(" ").delete(",").downcase
-  stripped == stripped.reverse
+def palindrome?(s)
+  forward = s.downcase.delete(" ").delete(",")
+  backward = s.downcase.reverse.delete(" ").delete(",")
+    if forward == backward
```

Comment:
this is too complex. You could just write "forward == backward" and that would return the correct value.